src/Lean/Data/Lsp/Basic.lean

```diff
@@ -44,6 +44,12 @@ structure Range where
   «end» : Position
   deriving Inhabited, BEq, Hashable, ToJson, FromJson
 
+instance : Ord Range where
```

Comment:
Doesn't `derive Ord` already behave this way? See #914

Reply:
Interesting, I wouldn't have thought that I changed the semantics of a related function when reordering the fields of structure.